packages/ckeditor5-package-generator/lib/utils/choose-programming-language.js

```diff
@@ -8,8 +8,8 @@
 const { prompt } = require( 'inquirer' );
 
 const PROGRAMMING_LANGUAGES = {
-	JavaScript: 'js',
-	TypeScript: 'ts'
+	JavaScript: { value: 'js', displayName: 'JavaScript' },
+	TypeScript: { value: 'ts', displayName: 'TypeScript (experimental)' }
```

Comment:
Shouldn't it be an array, then?

Reply:
I was a little bit worried at the beginning that changing it to an array might break something. But all works fine. Good catch!